ooniapi/services/ooniprobe/src/ooniprobe/routers/v1/probe_services.py

```diff
@@ -141,4 +174,17 @@ class ProbeUpdateResponse(BaseModel):
 @router.put("/update/{client_id}", tags=["ooniprobe"])
 def probe_update_post(probe_update: ProbeUpdate) -> ProbeUpdateResponse:
     log.info("update successful")
+    
+    # Log update metadata into prometheus
+    probe_update_dict = probe_update.model_dump(exclude_none=True)
+
+    # Info doesn't allows list, if we have a list we have to convert it 
+    # to string
+    if probe_update_dict['supported_tests'] is not None:
```

Comment:
Given that you have `exclude_none` above, won't this produce a key error if the probe didn't sent `supported_tests` or if it's set to `None`?

Maybe it's better to do `probe_update_dict.get('supported_tests')`?

Reply:
Yes you're right, thanks for noticing!